nodes/src/nodes/tool_git/IGlobal.py

```
@@ -0,0 +1,130 @@
+        self.repo = None
+        if self._tmp_dir:
+            shutil.rmtree(self._tmp_dir, ignore_errors=True)
+            self._tmp_dir = None
+
+
+def _is_url(value: str) -> bool:
+    """Return True if value looks like a remote git URL."""
+    return value.startswith(('http://', 'https://', 'git://', 'git@', 'ssh://'))
+
+
+def _parse_bool(raw: object, default: bool = True) -> bool:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)